### PR TITLE
458 fix wasm

### PR DESCRIPTION
**Pull request by matthew-russo**
replaces #458

### CODE REVIEW

**Comment by RReverser** [comment]
From a quick glance, looks good, although it's been open for a long time, so would be good to test it still works.

File: proptest/src/test_runner/config.rs [hunk 1]
```diff
@@ -10,51 +10,34 @@
...
-#[cfg(feature = "std")]
-const MAX_GLOBAL_REJECTS: &str = "PROPTEST_MAX_GLOBAL_REJECTS";
-#[cfg(feature = "std")]
-const MAX_FLAT_MAP_REGENS: &str = "PROPTEST_MAX_FLAT_MAP_REGENS";
-#[cfg(feature = "std")]
-const MAX_SHRINK_TIME: &str = "PROPTEST_MAX_SHRINK_TIME";
-#[cfg(feature = "std")]
-const MAX_SHRINK_ITERS: &str = "PROPTEST_MAX_SHRINK_ITERS";
-#[cfg(feature = "std")]
-const MAX_DEFAULT_SIZE_RANGE: &str = "PROPTEST_MAX_DEFAULT_SIZE_RANGE";
```
Comment (rexmas) [inline proptest/src/test_runner/config.rs:41]: did this one get deleted on accident? i believe it's still in use in `parse_or_warn`

Comment (RReverser) [reply proptest/src/test_runner/config.rs:41]: Idk - it's been a very long time - but CI seems to pass?

Comment (bmwill) [reply proptest/src/test_runner/config.rs:41]: Yeah this is fixed in https://github.com/proptest-rs/proptest/pull/518

Comment (RReverser) [reply proptest/src/test_runner/config.rs:41]: > so arbitrary tokens in pattern match positions seem to be treated as new variable bindings and default to (), that's why this still compiles even though there's a warning.

Hm but that doesn't explain why `parse_or_warn` accepted it if it's `()` and used in an argument that expects `&str`.

Comment (matthew-russo) [reply proptest/src/test_runner/config.rs:41]: pushed an updated commit that has this

**Review by rexmas — APPROVED** [review]
nice

**Pull request merged**